### DISCUSSION TITLE
不急 | /cn/2022/07/no-hurry/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1522] [Comments]
好嘛个杨阿姨
公然违抗我的三令五申
又备了生日礼
黄老师赶来　双手奉上
子恺散文漫画集
惊天一问：三十九？
才二十八岁　零一百二十个月的我
暗暗叹息
内心甚至几乎小声哭泣
却强装镇定
跺地一答：咳咳
不急不急
不着急
不用着急

https://yihui.org/cn/2022/07/no-hurry/

**Comment by liechi** [comment]
虽然不急，但能先剧透下今年是诗多还是文多吗？

**Reply by yihui** [reply]
都很少。去年下半年写了几篇，今年都没怎么写，所以今年很可能就不更新了……

**Comment by ZaiyanHere** [comment]
好巧，我的生日在7/10。

**Reply by yihui** [reply]
是有点巧，你跟我的一个妹妹同一天生日，每年我们都假惺惺礼尚往来式前后脚互祝一下生日快乐，哈哈。

**Reply by ZaiyanHere** [reply]
好啊好啊 就这么说定了

**Reply by yihui** [reply]
啊，误会我的意思了，我是说我跟我那个妹妹每年会假惺惺互祝一下。当然，如果以后你想每年生日在这里盖楼也是没问题的。

**Reply by yihui** [reply]
盖楼工人到此一游并祝生日快乐。

**Reply by ZaiyanHere** [reply]
生日快乐！我昨天刚从国内飞回来，倒时差中…：）
